conf/htconfig.sample.php

```diff
@@ -66,11 +66,11 @@
 
 App::$config['system']['register_policy'] = REGISTER_OPEN;
 App::$config['system']['register_text'] = '';
-App::$config['system']['admin_email'] = '';
+App::$config['system']['admin_email'] = '__EMAIL__';
 
 // Location of PHP command line processor
 
-App::$config['system']['php_path'] = 'php';
+App::$config['system']['php_path'] = 'php__PHPVERSION__';
```

Comment:
```suggestion
App::$config['system']['php_path'] = '/usr/bin/php__PHPVERSION__';
```
We may need PHP path here...